subsys/net/lib/lwm2m/Kconfig

```diff
@@ -111,7 +111,7 @@ config LWM2M_COAP_MAX_MSG_SIZE
 	default LWM2M_COAP_BLOCK_SIZE
 	help
 	  CoAP message size used by LWM2M. Minimum is the block size used
-	  in blockwise transfers.
+	  in blockwise transfers. Needs to be a multiple of the block size.
```

Comment:
I don't think I understand why do we need such a requirement. And I don't think it should actually be used the way it is, i.e. to determine the "large buffer" size. The option description is pretty clear - "CoAP message size used by LWM2M."

Reply:
This is not a strict requirement. I can remove that.